RELEASE.md

```diff
@@ -2,12 +2,6 @@
 
 In this document you'll find all the necessary steps to release a new version of `xcodeproj`:
 
-1. Re-generate the Carthage project with `tuist generate` *(Install [Tuist](https://github.com/tuist/tuist) if you don't have it installed already)*.
-2. Update Carthage dependencies if they are outdated with `bundle exec rake carthage_update_dependencies`.
-3. Validate the state of the project by running `bundle exec rake release_check`.
-4. Update the `CHANGELOG.md` adding a new entry at the top with the next version. Make sure that all the changes in the version that is about to be released are properly formatted.
-5. Update the version in the `xcodeproj.podspec` and `README.md` files.
-7. Commit, tag and push the changes to GitHub.
-8. Create a new release on [GitHub](https://github.com/tuist/XcodeProj) including the information from the last entry in the `CHANGELOG.md`.
-9. Push the changes to CocoaPods: `bundle exec pod trunk push --allow-warnings --verbose`.
-10. Archive the Carthage framework by running `bundle exec rake arhive_carthage` and attach the `XcodeProj.framework.zip` to the GitHub release.
+1. Run `tapestry release version-number` (eg `tapestry release 7.1.0`) *(Install [tapestry](https://github.com/ackeecz/tapestry) and [tuist](https://github.com/tuist/tuist) if you don't have them installed already)*.
```

Comment:
@fortmarek will Tapestry update the CHANGELOG to include the new section? In essence what used to be the step 4.

Reply:
Yes, it finds `## Next` and replaces it with 
```
## Next

## Version-number
```

Reply:
Nice :+1:. Well handled!

Reply:
It also updates the version in `README.md` and in `.podspec`. I ran a trial run locally and it seemed fine.